sdcm/nemesis.py

```diff
@@ -3683,6 +3683,12 @@ def disrupt_remove_node_then_add_node(self):  # pylint: disable=too-many-branche
         if self._is_it_on_kubernetes():
             raise UnsupportedNemesis("On K8S nodes get removed differently. Skipping.")
 
+        if SkipPerIssues('https://github.com/scylladb/scylladb/issues/21815', params=self.tester.params):
+            # TBD: To be removed after https://github.com/scylladb/scylladb/issues/21815 is resolved
+            ignore_issues_cm = ignore_stream_mutation_fragments_errors
+        else:
+            ignore_issues_cm = contextlib.nullcontext
+
```

Comment:
i suggest to use it before the context manager line number 711. it will be closer and clear to understand why this code is used.
Also `ignore_issues_cm` is not clear. better to rename `ignore(skip)_stream_mutation_errors_due_to_issue` or another meaningful name

Reply:
and fix pre-commits